packages/core/lib/commands/debug/run.js

```diff
@@ -1,14 +1,21 @@
 module.exports = async function (options) {
   const { promisify } = require("util");
   const debugModule = require("debug");
+  const inlineConfigNetwork = require("./inlineConfigNetwork");
   const debug = debugModule("lib:commands:debug");
 
   const { Environment } = require("@truffle/environment");
   const Config = require("@truffle/config");
 
   const { CLIDebugger } = require("../../debug");
 
-  const config = Config.detect(options);
+  let config = null;
```

Comment:
Assuming this logic is correct and we want this, a slick way to do this would be the following:
```
const config = options.url ? inlineConfigNetwork(options) : Config.detect(options);
```

Reply:
Also, minor style matter, but if you just want to create the `config` variable, it makes more sense to just write `let config;` rather than `let config = null;`.  Writing `let config = null;` suggests to the reader that the particular value `null` is somehow meaningful, when in fact it will never get used.